.changeset/proud-dolls-behave.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend': minor
+---
+
+Support file globbing in fs:delete
```

Comment:
add new words to `.github/vale/config/vocabularies/Backstage/accept.txt` as part of the pr to silence the spell checker

Reply:
Done